addons/product_images/models/res_config_settings.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    google_custom_search_key = fields.Char(
+        string="Google Custom Search API Key",
+        config_parameter='google.custom_search.key',
+    )
+    google_custom_search_cx = fields.Char(
```

Comment:
It's the name of the parameter on [Google's side](https://developers.google.com/custom-search/v1/reference/rest/v1/cse/list). I thought that It would be better to use the same name as Google to avoid confusion, but I understand that it may be misleading. Do you think that a name like `google_pse_id` would be better?

Reply:
Yes. The mapping to the param name on Google side will be done in the payload of the request anyway.